queries.sql

```diff
@@ -31,3 +31,56 @@ WHERE neutered = true;
 SELECT *
 FROM animals
 WHERE  weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Transactions and modifications
+
+BEGIN;
+UPDATE animals SET species = 'undefined';
+SELECT * FROM animals;
+ROLLBACK;
```

Comment:
Nice job writing this query :+1: You verified that the change was made before rolling back. You also need to verify that the `ROLLBACK` took effect. You can do so with another `SELECT` query after the `ROLLBACK`.